### PR TITLE
Kg/api format

### PR DESCRIPTION
**Pull request by kagarmoe**
Fixes a formatting issue.
Screen Shot 2020-02-10 at 4 37 50 PM

### CODE REVIEW

**Comment by phiggins** [comment]
This PR isn't going to pass CI because when you compiled the protos this file got deleted: `api/interservice/cfgmgmt/service/cfgmgmt.pb.client_mock.go`. I had that problem, and I was able to workaround it be deleting everything in the `bin/` directory of automate and recompiling the protos.

**Comment by kagarmoe** [comment]
Thanks for letting me know!

**Comment by phiggins** [comment]
You might also be able to fix it by checking that file back in (ie, undo the deletion) but it'll just keep happening when you compile the protos.

File: api/external/applications/applications.proto [hunk 1]
```diff
@@ -319,18 +319,14 @@ message ServicesReq {
...
-  // * `status:critical`: only return services that are returning a "critical"
-  //   healthcheck result
-  // * `status:unknown`: only return services that are returning an "unknown"
-  //   healthcheck result
-  // * `status:warning`: only return services that are returning a "warning"
-  //   healthcheck result
-  // * `status:ok`: only return services that are returning "ok" health check
-  //   results
+  // `status` filters refine the service results by a service's
+  //  most recent connected/disconnected state or healthcheck result.
```
[the remark this answers is not in the document]
Comment (kagarmoe) [reply api/external/applications/applications.proto]: How is this?
```
  // `status` filters refine service results by a service's
  //  current state or most recent healthcheck result.
  //  Disconnected services keep their last healthcheck result
  //  until their reports are removed by Chef Automate.
  //  When you apply a healthcheck filter, the report includes
  //  all recently disconnected services.
```

File: api/external/applications/applications.proto [hunk 2]
```diff
@@ -319,18 +319,14 @@ message ServicesReq {
...
+  //  Valid status filter parameters are:
```
[the remark this answers is not in the document]
Comment (kagarmoe) [reply api/external/applications/applications.proto]: Bulleted lists have their own special rules. It looks like spaces after paragraphs work automatically, but it is worth checking in dev.

**Comment by vjeffrey** [comment]
i told kimberly i would work on getting pr rebased and green on CI so we can merge. 
the commits are being real funky so i've decided to go the path of least resistance: grab the actual changes from this branch and start a new branch fresh from master with just those changes (proto) and recompile all the things. will hopefully have a new pr up soon

**Pull request closed without merging**